tests/functional/docker-compose.yml

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2015, 2016, 2017 CERN.
+#
+# hepcrawl is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+version: '2'
+
+services:
+  test_wsp:
+    image: hepcrawl_docker
+    environment: &env_variables
```

Comment:
Actually... tested this, and docker-compose seems not to be respecting the full yaml syntax, and any anchors I try to use (including yours) end up not not working on fedora with the latest packaged version of docker-compose, can you retry on your end?